ngraph/src/ngraph/op/fused/prelu.cpp

```diff
@@ -90,3 +91,58 @@ shared_ptr<Node> op::PRelu::clone_with_new_inputs(const OutputVector& new_args)
     }
     return make_shared<PRelu>(new_args.at(0), new_args.at(1));
 }
+
+template <element::Type_t ET>
+bool evaluate(const HostTensorPtr& arg0,
+              const HostTensorPtr& arg1,
+              const HostTensorPtr& out,
+              const op::AutoBroadcastSpec& broadcast_spec)
+{
+    runtime::reference::prelu(arg0->get_data_ptr<ET>(),
+                              arg1->get_data_ptr<ET>(),
+                              out->get_data_ptr<ET>(),
+                              arg0->get_shape(),
+                              arg1->get_shape(),
+                              broadcast_spec);
+    return true;
+}
+
+bool evaluate_prelu(const HostTensorPtr& arg0,
+                    const HostTensorPtr& arg1,
+                    const HostTensorPtr& out,
+                    const op::AutoBroadcastSpec& broadcast_spec)
+{
+    bool rc = true;
+    switch (arg0->get_element_type())
+    {
+        TYPE_CASE(i8)(arg0, arg1, out, broadcast_spec);
```

Comment:
I am not sure that we need PRelU for all these data_types.

Please remove all redundant data types in order to decrease impact on binary size.

Reply:
Since I'm also working on some evaluates implementation, I have a general question regarding which precisions are required and which have to be considered as redundant?

Reply:
redundant data types are data types which are not faced in our validation.
I understand that it is a bad metric. But it is a compromise between functionality and binary size.